Mandater/ClientApp/interfaces/states/ElectionState.ts

```diff
@@ -17,4 +17,26 @@ export interface ElectionState {
     levelingSeatsPlaceholder: number,
     partyResults: PartyResultDictionary,
     autoCompute: boolean;
-}
\ No newline at end of file
+}
+
```

Comment:
I feel like both states and the initial states should be moved into /store

Reply:
Why do they belong in /store rather than /states?

Reply:
I suppose creating a specific /state folder makes sense as well, though putting anything except interfaces in interfaces feels wrong -- we could also consider renaming "enums" to "types" and having a convention of any plural names being enums (so MonkeyData would be an interface or a type, and Monkeys would be an enum)

Reply:
I am for renaming enums to types, although I feel the definition of plural meaning a reference to an enum would be too vague. For example Data is both singular and plural.

I suggest the following change:
Moving the /states folder out of /interfaces
Renaming Enums to Types